### PR TITLE
I41 specify root path no results move yet - PART 1

### LINKED ISSUE #41: Specify root path for data output
blocks #9 
Specifying the root path allows for shifting the data output into a scratch location away from the git checkout. This location will then be used to publish the data out elsewhere.

Currently this will default to the existing location but potentially this should shift to default to ./data/generated and be added to the gitignore. This final move would also suggest shuffling the current .rds files from ./data to ./data/source or ./data/reference to help keep the data folder tidy.

### PR DESCRIPTION
**Pull request by joeHickson**
closes #41 without moving results location

### CODE REVIEW

**Comment by joeHickson** [comment]
Note: Rebase not merge with this branch.

**Comment by joeHickson** [comment]
This and it's sibling branch should be merged when the main box is not processing otherwise it will cause mayhem

**Comment by joeHickson** [comment]
It's sibling branch needs creating from scratch when the time is right as it can't handle the merge on .rds files annoyingly.
1) edit git ignore to include regional / national / subnational folders again and commit
2) git mv subnational data/results && git mv national data/results && git mv regional data/results and commit
3) revert 1 and commit

**Comment by joeHickson** [comment]
relates to #41

**Comment by joeHickson** [comment]
size won't help this merge conflict - it's because it moves them but trunk then gets updated by runs. Same with the last-update rds files. Rebase deals with it.

**Comment by seabbs** [comment]
Csv comment was because they can be versioned unlike rds. Sounds fine though. 

I think the readme here is out of date and needs less focus on developers and more on end data users (the warning is for people running estimates and that is not the target for the first paragraph of the readme. 

Other changes all look good to me. Just to confirm the default for now will be to publish estimates to the current path? 

Just to clarify are you suggesting rebasing to this branch? That doesn’t feel like a great idea given the changes on master.

**Comment by joeHickson** [comment]
No, rebasing this branch from master and then merging - it will then do the move to the latest version of the files and avoid a messy conflict.

I think the last runtime .rds file may need reviewing at some point anyway - we might want to switch to something local to the server rather than pushing them back to git. 

I'll need to resolve the readme when I next rebase - it should conflict with all the changes on trunk so I can update that

**Comment by seabbs** [comment]
Ah sorry right I understand. All sounds and looks good then.

Yes agree on last update not needed to be pushed now we have the status csv.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
+
+WARNING: We are in a transition phase - the results can be generated in two locations. Only results in /data/results will be committed and production will shift to this location. Data will be output to this location by using the `-d` flag. For those who require data in the old location you can (for the time being) run without the -d flag. This will be deprecated!
```
Comment (seabbs) [inline README.md:6]: Suggest drop notes about estimate generation as the intro paragraph should strictly target data users (as per #107) with developer info elsewhere + linked.

**Review by seabbs — APPROVED** [review]
Looks. Good see note about documentation structure as per discussion (#107).

File: README.md (hunk 1 above)
Comment (seabbs) [reply README.md:6]: Also this may now have a merge issue due to #104 and #90?

Comment (joeHickson) [reply README.md:6]: I'll do a rebase of the latest master and see what happens

**Comment by joeHickson** [comment]
rebase done. I'm aware this is a bit ugly but I think it:
a) tidies up some of the ancillary files into a nicer project structure
b) is an interim step I can't get around for putting the results into /data/results en-route to stopping publication of the results in github

I am tempted to can the whole branch though. When we get publishing off-box settled and then we can just cut the results entirely from the repo and then do the tidy of the few runtime / reference files as per this branch. No need for a flag to control output location - do I care if it isn't written to /data/results if it's in the git ignore? it certainly goes down the priority list somewhat...

**Comment by seabbs** [comment]
Close or move?

**Pull request closed without merging**